.travis.yml

```diff
@@ -15,8 +15,8 @@ install:
   - pip install --upgrade pip
   - pip install -r requirements.txt
   - pip install -r requirements-test.txt
-  - git clone https://github.com/StackStorm/st2.git --depth 1 --single-branch --branch v1.5.1 /tmp/st2 
-  # Super hacky work-around for the problem of the st2.tests.conf having a hard-coded path to the packs in it
-  - mkdir packs && ln -s $PWD packs/arteria-packs
+  - git clone https://github.com/StackStorm/st2.git --depth 1 --single-branch --branch v1.5.1 /tmp/st2
```

Comment:
Do you have to clone this repo here when you seem to be doing the same in `prepare_test_env.sh`?